Brokerages/GDAX/GDAXBrokerage.Messaging.cs

```diff
@@ -560,7 +560,7 @@ private void FillMonitorAction()
             }
             catch (Exception exception)
             {
-                OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Error, -1, exception.Message));
+                OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Warning, -1, exception.Message));
```

Comment:
Believe the monitoring task still stop running though? Maybe instead of `throw new Exception($"GDAXBrokerage.FillMonitorAction()....` above we could directly trigger this `Warning` message, not throw and continue, so it keeps running. And if there is another unexpected exception we **do** emit a `Error` message type here